islandora.module

```diff
@@ -2170,3 +2170,31 @@ function islandora_islandora_get_breadcrumb_query_predicates(AbstractObject $obj
     'fedora-rels-ext:isMemberOf',
   );
 }
+
+/**
+ * Implements hook_islandora_overview_object().
+ */
+function islandora_islandora_overview_object(AbstractObject $object) {
+  $cmodels = $object->models;
+  $rows = array();
+  foreach ($cmodels as $cmodel) {
+    if ($cmodel != 'fedora-system:FedoraObject-3.0') {
+      $o = islandora_object_load($cmodel);
+      if (is_object($o)) {
+        $rows[] = array(l($o->label, "islandora/object/{$o->id}"));
+      }
+      else {
+        $rows[] = array(t("@cmodel  - (This content model is not in this Islandora repository.)", array('@cmodel' => $cmodel)));
+      }
+    }
+  }
+  $header = count($rows) > 1 ? t('Content Models') : t('Content Model');
+  $table = array(
+    '#theme' => 'table',
+    '#header' => array($header),
+    '#rows' => $rows,
+    '#empty' => t('No Islandora content model associated with this object.'),
+  );
+  $rendered_table = drupal_render($table);
```

Comment:
It shouldn't be necessary to explicitly render things (admittedly, our documentation could use improvement). Just pass the renderable array, and it should get passed along and eventually rendered by Drupal, allowing opportunities to alter elements without touching markup.

For example: Say there was a desire to add attributes to various elements, then you could implement an alter ([either that specific to the overview page](https://github.com/Islandora/islandora/blob/7.x/islandora.module#L936), or possibly [Drupal's `hook_page_alter()`](https://api.drupal.org/api/drupal/modules%21system%21system.api.php/function/hook_page_alter/7.x), either in your theme or another module), and make whatever adjustments by adding to the array for your particular element(s).